src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/tests/test_keyvault_commands.py

```diff
@@ -601,7 +601,7 @@ class KeyVaultSoftDeleteScenarioTest(ResourceGroupVCRTestBase):
     def __init__(self, test_method):
         super(KeyVaultSoftDeleteScenarioTest, self).__init__(__file__, test_method,
                                                              resource_group='cli-test-kv-sd')
-        self.keyvault_name = 'cli-kv-test-softdelete'
+        self.keyvault_name = 'cli-kv-test-softdelete1'
```

Comment:
use a random name here to enable live test.

Reply:
I'm in the process of that. Should be done in the morning. So far so good.